packages/core/src/search/Search.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useRef, useState } from 'react';
+
+import Popup from './private/components/Popup';
+import { Result, SearchInputSize, SearchResults } from './private/components/SearchResults';
+import { SearchForm } from './private/components/Form';
+import {
+  SearchBox,
+  SearchClearIcon,
+  SearchInput,
+  SearchInputIcon,
+} from './private/components/Input';
+
+interface SearchProps {
+  buttonText?: string;
+  loadingMessage?: JSX.Element;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onSubmit?: (e: React.FormEvent<HTMLFormElement>) => void;
+  placeholder?: string;
```

Comment:
JSX.Element; as well

Reply:
I'm guessing placeholder cannot be JSX.Element, bc the it will go to `<input placeholder="..." />` which is string.